recipe-server/normandy/recipes/tests/test_checks.py

```diff
@@ -0,0 +1,48 @@
+from datetime import timedelta
+
+import pytest
+
+from normandy.recipes import checks, signing
+from normandy.recipes.tests import RecipeFactory, SignatureFactory
+
+@pytest.mark.django_db
+class TestSignaturesUseGoodCertificates(object):
```

Comment:
I'm not sure why I resisted writing tests for the checks before. This was a lot easier than I expected it to be.